### DISCUSSION TITLE
Can we have a new release please?

### DISCUSSION THREAD

**Discussion by forkwhilefork** [napalm-automation/napalm#1930] [General]
I ran into a bug that's already been fixed (#1743) but hasn't been released yet. It was fixed in September 2022, so 8 months ago. In March 2023 (>2 months ago) someone opened another issue about the same problem (#1862), not realizing it was already reported and fixed.

Can we cut a 4.0.1 release soon so that this bug (and others) can be fixed? I'd very much like to avoid editing installed pip packages on disk, especially inside of a docker container (which is how I'm running the software that's using NAPALM and hitting this bug).

**Comment by ktbyers** [comment]
We are working on it. We need to fix some issues on `get_bgp_config()`. The last two of these are probably here:

https://github.com/napalm-automation/napalm/pull/1925

and here:

https://github.com/napalm-automation/napalm/pull/1926

**Comment by forkwhilefork** [comment]
4.1.0 was released today. Thanks!
